core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -688,6 +688,12 @@ public enum CoreError implements ScalarDbError {
       "Invalid number specified for column %s in table %s in namespace %s",
       "",
       ""),
+  ABAC_NOT_ENABLED(
+      Category.USER_ERROR,
+      "0151",
+      "The Attribute-Based Access Control feature is not enabled. To use this feature, you must enable it. Note that this feature is supported only in the ScalarDB Enterprise edition",
+      "",
+      ""),
```

Comment:
@josh-wong Could you please take a look at this message? 🙇

Reply:
Similar to many of our other features (group commit, encryption, etc.), should we lowercase the naming for this feature as follows?:

```suggestion
  ABAC_NOT_ENABLED(
      Category.USER_ERROR,
      "0151",
      "The attribute-based access control feature is not enabled. To use this feature, you must enable it. Note that this feature is supported only in the ScalarDB Enterprise edition",
      "",
      ""),
```

Reply:
Fixed in 11f38dbf38fd216b4ff005fe160c5fbd242ccc4f. Thanks!